fabric_plugin/tunnel.py

```diff
@@ -42,22 +39,27 @@ def accept(channel, (src_addr, src_port), (dest_addr, dest_port)):
         try:
             sock.connect((local_host, local_port))
         except Exception as e:
+            try:
+                channel.close()
+            except Exception as e2:
+                close_error = ' (While trying to close channel: {0})'.format(
```

Comment:
I'm wondering.. did you stumble upon a situation in which tie channel failed to close or are you just taking the safe approach?